impexp-client/src/main/java/org/citydb/modules/kml/gui/preferences/GeneralPanel.java

```diff
@@ -79,6 +81,8 @@
 	private JButton gltfConverterBrowseButton = new JButton("");
 	private JCheckBox notCreateColladaCheckbox = new JCheckBox();
 	private JCheckBox embedTexturesInGltfCheckbox = new JCheckBox();
+	private JCheckBox exportGltfBinary = new JCheckBox();
+	private JCheckBox enableGltfDracoCompression = new JCheckBox();
```

Comment:
Moved to branch [glb-draco-support](https://github.com/3dcitydb/importer-exporter/tree/glb-draco-support).